controlplane/pkg/nsmd/serviceregistry.go

```diff
@@ -199,6 +199,8 @@ func (impl *nsmdServiceRegistry) initRegistryClient() {
 			continue
 		}
 		impl.registryClientConnection = conn
+		for ; conn.GetState() != connectivity.Ready; <-time.Tick(300 * time.Millisecond) {
```

Comment:
Should we have similar code in all places we init GRPC connection? also should we have smaller tick timeout?

Reply:
I remove waiting for Ready state and can't reproduce problem anymore. According to documentation there is no need to wait particular state

Reply:
If we find this to be a common code, we should put it in a common `utility` or `tools` file.